examples/Demo/FluentUI.Demo.Client/Documentation/Components/CompoundButton/Examples/CompoundButtonDefault.razor

```diff
@@ -0,0 +1,34 @@
+﻿<p>
```

Comment:
The razor files must contain only the source code, not the documentation.
The documentation must be included in the `FluentCompoundButton.md` file.

Reply:
Including the sentence about label/childcontent in the doc